### DISCUSSION TITLE
valvole termostatiche tado

### DISCUSSION THREAD

**Discussion by lele-65** [indomus/forum#1122] [HUB: Home Assistant]
Buongiorno
ho montato le valvole termostatiche TADO, funzionano bene ma vorrei riuscire a regolare l'apertura in base alla temperatura della stanza rilevata da sensore zigbee. E' possibile?
Grazie

**Comment by snakuzzo** [comment]
In teoria potresti farlo, ma non è così banale da realizzare.
Questo perché le singole teste termostatiche si regolano in base alla propria temperatura, non a quella di sensori esterni.
Potresti regolare l'offset "fisso" su singola testa, ma non è detto che basti, anzi...quasi sicuramente non basterà.
Quindi dovresti inventarti un'automazione/script che ad ogni rilevazione del sensore esterno regoli l'offset in automatico in base alla differenza tra la temperatura rilevata dal sensore zigbee e quella rilevata dalla testa Tado. Prova a dare un'occhiata [QUI](https://community.home-assistant.io/t/tado-integration-put-offset/175964/16)

**Reply by lele-65** [reply]
grazie ci guardo
la mia idea era di poter impostare direttamente una percentuale di apertura, bypassando la logica della valvola. Fare quindi una regolazione PI esternamente alla valvola

**Reply by snakuzzo** [reply]
purtroppo non puoi bypassare la logica della testa termostatica (non della valvola)

**Comment by 50494554524F** [comment]
metti offset a -9, quando le apri (devi farlo sempre ogni volta) imposta la temperatura a 25... in questo modo puoi simulare molto bene apri e chiudi e bypassare il firmware... se ti accorgi che la valvola si scalda e chiude abbassa un po' la mandata

personalmente, con queste impostazioni e mandata a 55° non ho problemi...

**Reply by snakuzzo** [reply]
Sì potrebbe farlo. Io ho usato questo sistema per mia moglie per farle asciugare i panni umidi :D
...utilizzando le API di Tado e mettendo l'offset a `-9.9 °C` creando degli degli switch template.

In pratica quando serve questa sorta di asciugatrice...accendo lo switch...che
- mi imposta l'offset -9.9
- mi imposta il climate a 25°C e va in manuale (bypassando la schedulazione fatta da app)

Quando invece non mi serve più...spengo lo switch...che
- mi imposta l'offset a 0 °C
- mi imposta il climate con preset auto (riutilizzando nuovamente la schedulazione fatta da app)

Lui potrebbe applicare la stessa logica tenendo fisso l'offset a -9.9°C

![asc](https://user-images.githubusercontent.com/7205371/144277332-ef1ec195-39ad-40a4-b7ea-1585a50af459.png)
